hrms/hr/report/shift_attendance/shift_attendance.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2023, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.query_reports["Shift Attendance"] = {
+  filters: [
+    {
+      fieldname: "from_date",
+      label: __("From Date"),
+      fieldtype: "Date",
+    },
+    {
+      fieldname: "to_date",
+      label: __("To Date"),
+      fieldtype: "Date",
```

Comment:
Let's make these filters mandatory and set them to this month's start & end dates by default. Generally, attendance data volume is huge. It's generated every day so we might end up reading the entire table which is not good performance-wise. Also, people usually check such reports on a monthly basis (before payroll maybe so defaulting it to month start & end makes sense)

Also, make company filter mandatory and set it to the default company